services/catalog/src/simcore_service_catalog/core/settings.py

```diff
@@ -23,6 +24,17 @@ def base_url(self) -> str:
         return f"http://{self.DIRECTOR_HOST}:{self.DIRECTOR_PORT}/{self.DIRECTOR_VTAG}"
 
 
+_DEFAULT_SERVICE_RESOURCES = ServiceResources.parse_obj(
+    {
+        "CPU": {"limit": 0.1, "reservation": 0.1},
+        "RAM": {
+            "limit": ByteSize(2 * 1024**3),
+            "reservation": ByteSize(2 * 1024**3),
```

Comment:
Wait, so by default you get 2 GB limit and reservation? Are we sure about this? Should we not bring the values in the MB ranges?

For example, when defining services like the sim4life. You have an Nginx without any constraints. It is not worth it to assign so many resources for a service which requires  almost 100 MB.

I'd suggest to bump it down to 100MB (or something in this range), and let people explicitly ask for more resources if required. I think this will help with reducing memory consumption by a lot.

Reply:
ok, so that is the value that was here already before. Open to discussion but in a separate PR cause it might have other implications. That is what the current director-v0 also does (but not wired to this one for now), so it could have issues with some older services.

Now concerning your use case:
- this call returns the total envelope, not the nginx service. so for computing the nginx service in your compose it should go with: get service limit/resources - what is defined in the compose for specifics = nginx (+ maybe other) services limits